app/templates/administrative-units/administrative-unit/sites/site.hbs

```diff
@@ -0,0 +1,11 @@
+{{page-title @model.site.address.fullAddress}}
+
+{{breadcrumb
+  this.model.site.address.fullAddress
+  route="administrative-units.administrative-unit.sites"
+  models=(array this.model.site.id this.model.administrativeUnit.id)
```

Comment:
I think the order needs to be reversed here: admin unit id first, site id second. I think ember applies them from left to right.